### PR TITLE
Add search e2e test and updated packages

### PR DESCRIPTION
**Pull request by rajeshg**
- e2e test for search
- update packages

## Checklist

- [x] Tests updated
- [ ] Docs updated

### CODE REVIEW

File: tests/e2e/settings-profile.test.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ test('Users can update their profile photo', async ({ login, page }) => {
 	await expect(page).toHaveURL(`/settings/profile/photo`)
 
 	await page
-		.getByLabel(/change/i)
```
Comment (rajeshg) [inline tests/e2e/settings-profile.test.ts:67]: This is flaky and results in "node is not an htmlinputelement", guess file input type 
 nodes are not treated the same as text input nodes.

**Comment by rajeshg** [comment]
This test assumes that the default user "kody" always exists. Is it safe to make that assumption?

**Comment by kentcdodds** [comment]
Thank you!

**Comment by rajeshg** [comment]
This is definitely better. Thanks for improving this further

**Pull request merged**